src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,35 @@
-import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    body,
+    name,
+    email,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title is-flex is-align-items-center">
+        <strong className="CommentInfo__name mr-1">
+          {name}
+        </strong>
+
+        by
+
```

Comment:
do you need those empty lines here?

Reply:
I forgot to remove this one, but to be honest I like to visually decompose imports that have different purposes, but when we have only two imports maybe it's looks redundant